### DISCUSSION TITLE
reason: Client network socket disconnected before secure TLS connection

### DISCUSSION THREAD

**Discussion by rodrigoimenes** [quirrel-dev/quirrel#1027] [Q&A] [answered]
Hey guys! Im following the self-hosting doc https://dev.to/remixtape/self-hosting-quirrel-5af7.

I have an app on Heroku that im trying to connect with my instance of quirrel, but when i run `npx quirrel ci` i get this error:

![image](https://user-images.githubusercontent.com/8990835/184047648-d945f086-d907-4eeb-ab52-a5801b186040.png)

Im setting the `QUIRREL_API_URL` as `https://quirrel-app.fly.dev`, my token and `QUIRREL_BASE_URL` as `https://xpto.herokuapp.com` (xpto is not my app, but i dont want to share the real name 😅

**Comment by Skn0tt** [comment]
When I run `curl https://quirrel-app.fly.dev/queues/update-cron` locally, I can connect successfully to that Quirrel instance. Not sure what the error here is, hard for me to assist on this 🤔

**Comment by rodrigoimenes** [comment, marked as answer]
I've tried again and worked well, so i think it was something on heroku machine (maybe?)
Thanks!!! :D

**Reply by Skn0tt** [reply]
networking issues are always vast and mysterious ✨  ;D good to hear it works now!
